pkg/web/http.go

```diff
@@ -7,6 +7,7 @@ import (
 
 	"github.com/charmbracelet/log"
 	"github.com/charmbracelet/soft-serve/pkg/config"
+	"github.com/rs/cors"
```

Comment:
Please use https://github.com/gorilla/handlers

Reply:
will do. is it okay to set default CORS to "*"? do you think we need additional config?

Reply:
It would be nice to have this configurable with a default value set of `{HTTPConfig.PublicURL}/*`